### PR TITLE
Grafana version upgrade to 8.5.9

### PR DESCRIPTION
**Pull request by pjosephraj**
Grafana version upgraded to 8.5.9
- Branding changes
- Default theme to light

### CODE REVIEW

File: public/views/index-template.html [hunk 1]
```diff
@@ -180,7 +180,7 @@
           <div class="preloader__logo"></div>
         </div>
       </div>
-      <div class="preloader__text">Loading Grafana</div>
+      <div class="preloader__text">Loading Insights</div>
       <div class="preloader__text preloader__text--fail">
         <p>
           <strong>If you're seeing this Grafana has failed to load its application files</strong>
```
Comment (santosh-at-smarthub) [inline public/views/index-template.html:186]: `<strong>If you're seeing this Grafana has failed to load its application files</strong>`
I think this Grafana text replacement is missed with Insights.

**Review by santosh-at-smarthub — COMMENTED** [review]
LGTM. Please check some minor changes needed which highlighted in comment

File: public/views/index-template.html (hunk 1 above)
Comment (santosh-at-smarthub) [reply public/views/index-template.html:186]: Should we worry about half checks failed sir?

**Review by Raanjeetsgolu — APPROVED** [review]

**Review by Raanjeetsgolu — COMMENTED** [review]
some checks are failing. please look at those.

File: public/views/index-template.html (hunk 1 above)
Comment (AjithRavindranathan) [reply public/views/index-template.html:186]: Those are the grafana jobs which may not work for us in this fork

**Pull request merged**